src/stores/userStore.ts

```diff
@@ -42,6 +42,12 @@ export class UserStore implements IUserStore {
     telescoperApi.setToken(token)
   }
 
+  @action.bound
+  logout() {
+    this.setToken(undefined)
+    window.location.replace("/auth/login");
```

Comment:
I just realized during my testing, that because of the redirect code already in AppRoot.tsx, we don't need this line at all.
setToken changes the token which causes AppRoot to rerender, which notices that token is undefined, and redirects to /auth/login.

Reply:
Yep! Fixed @orndorffgrant . This is an even better way to handle it because the browser redirect is not held up but the response to the alert call. Thanks